server/app.js

```diff
@@ -423,10 +433,21 @@ if(isLocalEnvironment){
 const templateFn = require('./../client/template.js');
 const renderPage = async (req, res)=>{
 	// Create configuration object
+	const languageOptions = ['pt-BR','en-US', 'es-ES', 'fr-FR'];
+	const langCodeCorrect =()=> {
+		if (languageOptions.indexOf(req.language) > -1) {return req.language;}
+		return 'en-US';
+	};
+
+	const langPreference = langCodeCorrect();

```

Comment:
I would probably just simplify this to

```suggestion
	const langPreference = languageOptions.includes(req.language) ? req.language : 'en-US';
```